src/AniListNet/Objects/Media/Review/MediaReviewSort.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Runtime.Serialization;
+
+namespace AniListNet.Objects;
+
+public enum MediaReviewSort
+{
+    [EnumMember(Value = "ID")] Id,
+    [EnumMember(Value = "ID_DESC")] IdDesc,
+    [EnumMember(Value = "SCORE")] Score,
+    [EnumMember(Value = "SCORE_DESC")] ScoreDesc,
+    [EnumMember(Value = "RATING")] Rating,
+    [EnumMember(Value = "RATING_DESC")] RatingDesc,
+    [EnumMember(Value = "CREATED_AT")] CreatedAt,
+    [EnumMember(Value = "CREATED_AT_DESC")] CreatedAtDesc,
+    [EnumMember(Value = "UPDATED_AT")] UpdatedAt,
+    [EnumMember(Value = "UPDATED_AT_DESC")] UpdatedAtDesc,
```

Comment:
Should create a filter class like `GetMediaReviewsFilter` and include a property `SortDescending` and remove the `_DESC` properties. This is what I've done for searching [media](https://github.com/dentolos19/AniListNet/blob/394f31e9b872d832e9266859ffbd09c04921cf7c/src/AniListNet/Parameters/SearchMediaFilter.cs#L19) and [characters](https://github.com/dentolos19/AniListNet/blob/394f31e9b872d832e9266859ffbd09c04921cf7c/src/AniListNet/Parameters/SearchCharacterFilter.cs#L11).

I'm open for suggestions if this mechanism needs to be changed.

Reply:
Sounds good, I'll update that. 

I'm having a bit of issues with the mutations, I might need to either, split into another branch, or split to a branch and get some pointers from you.